docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -12,9 +12,15 @@
 The Core ML Tools [Unified Conversion API](unified-conversion-api) produces Core ML models for iOS 13, macOS 10.15, watchOS 6, tvOS 13 or newer deployment targets. If your primary deployment target is iOS 12 or earlier, you can find limited conversion support for PyTorch models via the [onnx-coreml](https://github.com/onnx/onnx) package.
 ```
 
-## Generate a TorchScript Version
+To export a model from PyTorch to Core ML, there are 2 steps:
+1. Capture the PyTorch model graph from the original torch.nn.Module, via [`torch.jit.trace`](https://pytorch.org/docs/stable/generated/torch.jit.trace.html) or [`torch.export.export`](https://pytorch.org/docs/stable/export.html#torch.export.export).
+2. Convert the PyTorch model graph to Core ML, via the Core ML Tools [Unified Conversion API](unified-conversion-api).
 
-[TorchScript](https://pytorch.org/docs/stable/jit.html) is an intermediate representation of a PyTorch model. To generate a TorchScript representation from PyTorch code, use PyTorch's JIT tracer ([`torch.jit.trace`](https://pytorch.org/docs/stable/generated/torch.jit.trace.html)) to _trace_ the model, as shown in the following example:
+Core ML Tools has been supporting the conversion from torch.jit.trace graph since Core ML Tools 4, so this the more mature and stable path. The conversion from torch.export graph is newly added in Core ML Tools 8.0, currently in beta state, in line with the export API status in PyTorch. As of Core ML Tools 8.0, representative models such as mobile bert, resnet, vit, [mobile net](convert-a-torchvision-model-from-pytorch), [deep lab](convert-a-pytorch-segmentation-model), [openelm](convert-openelm) can be converted, and total torch op translation test coverage is roughly ~60%. We would recommend to start trying the torch.export path, as [PyTorch no longer offers new feature development for torch.jit.trace](https://github.com/pytorch/pytorch/issues/103841#issuecomment-1605017153).
```

Comment:
---> (capital case for the first letters)
"
MobileBert, ResNet, ViT, [MobileNet](convert-a-torchvision-model-from-pytorch), 
[DeepLab](convert-a-pytorch-segmentation-model), [OpenELM](convert-openelm) etc can be converted
"


Why links for some models and no links for mobilebert, resnet, vit? If you are putting links, put them for all the models you mention.

Reply:
"Core ML Tools has been supporting the conversion from torch.jit.trace graph since Core ML Tools 4, so this the more mature and stable path. The conversion from torch.export graph is newly added in Core ML Tools 8.0, currently in beta state, in line with the export API status in PyTorch. As of Core ML Tools 8.0, representative models such as mobile bert, resnet, vit, [mobile net](convert-a-torchvision-model-from-pytorch), [deep lab](convert-a-pytorch-segmentation-model), [openelm](convert-openelm) can be converted, and total torch op translation test coverage is roughly ~60%. We would recommend to start trying the torch.export path, as [PyTorch no longer offers new feature development for torch.jit.trace](https://github.com/pytorch/pytorch/issues/103841#issuecomment-1605017153)." 


---->

"
The conversion from a graph captured via `torch.jit.trace` has been supported for many versions of coremltools, hence it is the stable and the more performant path. For now, this is the recommended way for converting PyTorch models to Core ML.  

The conversion from `torch.export` graph has been newly added to Core ML Tools 8.0. 
It is currently in beta state, in line with the export API status in PyTorch. 
As of Core ML Tools 8.0, representative models such as MobileBert<link>, ResNet<link>, ViT<link>, [MobileNet](convert-a-torchvision-model-from-pytorch), 
[DeepLab](convert-a-pytorch-segmentation-model), [OpenELM](convert-openelm) etc can be converted, and the total Torch op translation test coverage is roughly ~60%. 
You can start trying the torch.export path on your models that are working with torch.jit.trace already, so as to gradually move them to the export path as PyTorch also [moves](https://github.com/pytorch/pytorch/issues/103841) its support and development to that path over a period of time. In case you hit issues (e.g. models converted via export path are slower than the ones converted from jit.trace path), please report them on Github. "

Reply:
> Why links for some models and no links for mobilebert, resnet, vit? If you are putting links, put them for all the models you mention.

Because the 3 links are to our conversion examples, not to the model definitions.

> Breaking into 2 paragraphs

Broke as suggested